engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/settings/SettingsMenuScreen.java

```diff
@@ -41,6 +28,9 @@ public void initialise() {
         WidgetUtil.trySubscribe(this, "video", button -> triggerForwardAnimation(VideoSettingsScreen.ASSET_URI));
         WidgetUtil.trySubscribe(this, "audio", button -> triggerForwardAnimation(AudioSettingsScreen.ASSET_URI));
         WidgetUtil.trySubscribe(this, "input", button -> triggerForwardAnimation(InputSettingsScreen.ASSET_URI));
+        WidgetUtil.trySubscribe(this, "test", button -> triggerForwardAnimation(new ResourceUrn("engine:autoConfigTestScreen")));
+        WidgetUtil.trySubscribe(this, "test2", button -> triggerForwardAnimation(new ResourceUrn("engine:mathTypeWidgetTestScreen")));
+        WidgetUtil.trySubscribe(this, "test3", button -> triggerForwardAnimation(new ResourceUrn("engine:builtinTypeWidgetTestScreen")));
```

Comment:
Do we want to put this here, visible in the settings menu where players may also access it? I'd rather just instantiate and view these test screens via the console command (and not have them be accessible via the UI).

Reply:
Sounds good. There is a console command to create and display any UI screen (I forgot which one).